Directory.Build.props

```diff
@@ -0,0 +1,46 @@
+<Project>
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <Product>MQTTnet</Product>
+        <Authors>The contributors of MQTTnet</Authors>
+        <Company>The contributors of MQTTnet</Company>
+        <Copyright>Copyright (c) .NET Foundation and Contributors</Copyright>
+
+        <DelaySign>false</DelaySign>
+        <SignAssembly>false</SignAssembly>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <EnableNETAnalyzers>false</EnableNETAnalyzers>
+        <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
+
+        <NeutralLanguage>en-US</NeutralLanguage>
+        <NoWarn>1591;NETSDK1138;NU1803;NU1901;NU1902</NoWarn>
+
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <NuGetAuditMode>all</NuGetAuditMode>
+        <NuGetAudit>true</NuGetAudit>
+        <NuGetAuditLevel>low</NuGetAuditLevel>
+        <AnalysisLevel>latest-Recommended</AnalysisLevel>
```

Comment:
In my IDE, the latest-Recommended value causes hundreds of syntax errors.

Reply:
https://developercommunity.visualstudio.com/t/Code-Rules-that-were-Informational-are-n/10789409#TPIN-N10795393